x-pack/plugins/rule_registry/common/search_strategy/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ValidFeatureId } from '@kbn/rule-data-utils';
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { IEsSearchRequest, IEsSearchResponse } from 'src/plugins/data/common';
+import { ParsedTechnicalFields } from '../';
+
+export type RuleRegistrySearchRequest = IEsSearchRequest & {
+  featureIds: ValidFeatureId[];
+  query?: { bool: estypes.QueryDslBoolQuery };
+};
+export type RuleRegistrySearchResponse = IEsSearchResponse<ParsedTechnicalFields>;
```

Comment:
Don't use `ParsedTechnicalFields`. Use [Ecs](https://github.com/elastic/kibana/blob/main/packages/kbn-logging/src/ecs/index.ts#L65)

Reply:
yes we should use the Search fields API https://www.elastic.co/guide/en/elasticsearch/reference/current/search-fields.html#search-fields-response something like that `Record<string, any>`

Reply:
done!